src/hotspot/cpu/x86/x86.ad

```diff
@@ -5797,6 +5806,49 @@ instruct signumD_reg(regD dst, regD zero, regD one, rRegP scratch, rFlagsReg cr)
   ins_pipe( pipe_slow );
 %}
 
+#ifdef _LP64
+instruct copySignF_reg(regF dst, regF src, regF tmp1, rRegL tmp2) %{
+  predicate(UseAVX >= 3 && VM_Version::supports_avx512vl());
+  match(Set dst (CopySignF dst src));
+  effect(TEMP tmp1, TEMP tmp2);
+  format %{ "CopySignF $dst, $src" %}
```

Comment:
Please add the tmp registers to the format, something like below:
format %{ "CopySignF $dst, $src\t! using $tmp1 and $tmp2 as TEMP" %}